templates/admin/import.php

```diff
@@ -8,6 +8,13 @@
 $import_revoke_url = wp_nonce_url( get_admin_url( get_current_blog_id(), '/tools.php?page=pb_import&revoke=yes' ), 'pb-revoke-import' );
 $current_import = get_option( 'pressbooks_current_import' );
 $custom_post_types = apply_filters( 'pb_import_custom_post_types', array() );
+$import_option_types = apply_filters( 'pb_select_import_type', array(
+	'wxr' => 'WXR (WordPress eXtended RSS)',
```

Comment:
The values here should be wrapped in `__()`, e.g. `__( 'WXR (WordPress eXtended RSS)' )`. This will fix the Travis error, because…